lib/api/controllers/serverController.js

```diff
@@ -99,18 +123,18 @@ function ServerController (kuzzle) {
         services: {}
       };
 
-    Object.keys(kuzzle.funnel.controllers).forEach(controller => {
+    Object.keys(this.kuzzle.funnel.controllers).forEach(controller => {
       const actionList = {};
 
-      Object.keys(kuzzle.funnel.controllers[controller]).forEach(action => {
-        var routeDescription = {};
+      Object.keys(this.kuzzle.funnel.controllers[controller]).forEach(action => {
+        let routeDescription = {};
 
-        if (typeof kuzzle.funnel.controllers[controller][action] === 'function') {
+        if (typeof this.kuzzle.funnel.controllers[controller][action] === 'function') {
           actionList[action] = {name: action};
 
           // resolve associated http route for each actions
-          Object.keys(kuzzle.config.http.routes).forEach((key) => {
-            var route = kuzzle.config.http.routes[key];
+          Object.keys(this.kuzzle.config.http.routes).forEach(key => {
+            const route = this.kuzzle.config.http.routes[key];
 
             if ((route.controller === controller || controller === 'memoryStorage' && route.controller === 'ms') && route.action === action) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 142 characters long line (which is greater than 80 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ALineLength)